src/Microsoft.PowerShell.Commands.Utility/commands/utility/convert-HTML.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
     ///
-    /// Class comment
+    /// Class comment.
```

Comment:
Remove the empty summary comment lines as you did with other files.

Reply:
Fixed.